client/jetpack-cloud/sections/partner-portal/license-product-card/style.scss

```diff
@@ -4,6 +4,11 @@
 .license-product-card {
 	width: 100%;
 
+	&.disabled {
+		opacity: 0.4;
```

Comment:
As per my observations, the opacity we have set throughout the agency dashboard project is `0.5`. I would suggest sticking to `0.5`. 

One more thing we can do here is to set the cursor to `not-allowed` for `.license-product-card__inner`

So, we have to make it like 

```
&.disabled {
		opacity: 0.5;

		.license-product-card__inner {
			cursor: not-allowed;
		}
	}
```

and handle disabling selection in the `onSelect` by doing 

```
if ( isDisabled ) {
	return;
}
```